drivers/gke/gke_driver.go

```diff
@@ -699,6 +702,22 @@ func (d *Driver) waitCluster(ctx context.Context, svc *raw.Service, state *state
 	}
 }
 
+func (d *Driver) waitClusterRemoveExp(ctx context.Context, svc *raw.Service, state *state) (*raw.Operation, error) {
+	var operation *raw.Operation
+	var err error
+
+	for i := 1; i < 12; i++ {
+		time.Sleep(time.Duration(i*i) * time.Second)
+		operation, err := svc.Projects.Zones.Clusters.Delete(state.ProjectID, state.Zone, state.Name).Context(ctx).Do()
+		if err == nil {
```

Comment:
I think we need to handle errors when they are not the `wait and try again` error too.  Please add a case to handle that to the If statement.

Reply:
Disregard previous comment.